src/app/components/goal-dashboard/goal-dashboard.component.html

```diff
@@ -1,122 +1,131 @@
 <div class="main">
     <h1>{{username}}'s' goal dashboard!</h1>
-    <div class = "calander">
-        <p>You cannot choice more than 5 exercizes per session! <br> 
+        <p class = "info">You cannot choice more than 5 exercizes per session! <br> 
             you must choose all excersizes before you start looging them as finished!<br>
             One workout has five excersizes!
         </p>
-        <ol>
-            <li>
-                <span>
-                    {{date.getFullYear()+"-"+date.getMonth()+"-"+date.getDate()}}
-                </span>
-                <button id="commit" (click)="commitFinish()">Commit finishes!</button>
-                <span>
-                    <h3>Morning</h3>
-                    <span>
-                        <h4>Planned!</h4>
-                        <ul class="Excersizedisplay1">
-                            <li *ngFor="let choise of ex_choice_list; let i = index" >
-                              {{choise}}
-                              <button id="i" (click)="updateExFinish(this.i)">Finished!</button>
-                              <button id="i" (click)="remove(this.i)">Remove!</button>
-                            </li>>
-                        </ul>
-                    </span>
-                    <span>
-                        <h4>Finished!</h4>
-                        <ul class="ExFinishdisplay1">
-                            <li *ngFor="let finish of ex_finish_list; let i = index" >
-                              {{finish}}
-                              <button class="NotFinish" id="i" (click)="updateExPlaned(this.i)">Not finished!</button>
-                            </li>>
-                        </ul>
-                    </span>
-                    <select class = "ExcerciseOptions1" name="select1.1" (change)="onChangeEx()">
-                        <option *ngFor="let excersize of excersize_options ; let i = index" >
-                          {{excersize}}
-                        </option>
-                    </select>
+    
+        <div class="time">
+            <span>
+                {{date.getFullYear()+"-"+date.getMonth()+"-"+date.getDate()}}
+            </span>
+        </div>
+             
+        <div class = "calander">
+        <div class = "morning">
+            <h3>Morning</h3>
+            <span>
+            <h4>Planned!</h4>
+                <ul class="Excersizedisplay">
```

Comment:
Excersizedisplay = ExerciseDisplay
```suggestion
                <ul class="ExerciseDisplay">
```